build/configs/rtl8721csm/tc/defconfig

```diff
@@ -552,6 +557,11 @@ CONFIG_UART2_2STOP=0
 # Wireless Device Options
 #
 CONFIG_DRIVERS_WIRELESS=y
+CONFIG_DRIVERS_BLUETOOTH=y
+# CONFIG_DRIVERS_BR_EDR is not set
+CONFIG_DRIVERS_BLE=y
+# CONFIG_VIRTUAL_BLE is not set
++CONFIG_RTK_BLE=y
```

Comment:
Please remove '+'

Reply:
I think CONFIG_VIRTUAL_BLE is needed to run blemgr_tc instead of CONFIG_RTK_BLE, right?

Reply:
Something got complicated in Kconfig.
I recommend you to remove the content of 'os/drivers/bluetooth/virtual_ble/Kconfig'.
It always makes 'CONFIG_VIRTUAL_BLE' be disabled.

Reply:
By default CONFIG_VIRTUAL_BLE is N(disabled). Whenever you have to run the testcases you can enable from menuconfig.

Reply:
> Please remove '+'

Made changes